### PR TITLE
[Perl6/en]Modify an error about the Range constructor

### PR DESCRIPTION
**Pull request by kwl01skz**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [ ] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [ ] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: perl6.html.markdown [hunk 1]
```diff
@@ -501,9 +501,12 @@ say False ~~ True;  #=> True
 ##------------------
 3 .. 7;          # 3 to 7, both included
 3 ..^ 7;         # 3 to 7, exclude right endpoint.
-3 ^.. 7;         # 3 to 7, exclude left endpoint. Same as `4..7`.
-3 ^..^ 7;        # 3 to 7, exclude both endpoints. Same as `4..6`.
-
+3 ^.. 7;         # 3 to 7, exclude left endpoint.
+3 ^..^ 7;        # 3 to 7, exclude both endpoints.
+		 # 3 ^.. 7 almost like `4..7`. But when we consider decimals:
+3.5 ~~ 3 ^.. 7	 # True, This Range also contains decimals greater than 3.
```
Comment (divayprakash) [inline perl6.html.markdown]: Evaluates to ```false```

**Comment by kwl01skz** [comment]
> Please explain the change you've made here.

The original is wrong:
>  3 ^.. 7;         # 3 to 7, exclude left endpoint. Same as `4..7`.
> 3 ^..^ 7;        # 3 to 7, exclude both endpoints. Same as `4..6`.

3 ^.. 7 is not exactly the same as 4..7
My revision explains the difference.

Or we can do this
`say 3 ^.. 7 ~~ 4..7;  # =>false`

File: perl6.html.markdown (hunk 1 above)
Comment (kwl01skz) [reply perl6.html.markdown]: I got True.
If you get a False , maybe you have encountered a bug.
3.5∈(3,7] . This is a  mathematical fact.

**Comment by divayprakash** [comment]
@kwl01skz Thank you for the contribution and welcome to the LXIYM community!

**Pull request merged**